### DISCUSSION TITLE
RUIZU用(「日付_講座名」)のプリセット追加 、他

### DISCUSSION THREAD

**Discussion by CSReviser** [CSReviser/CaptureStream2#9] [Announcements]
＜全OS共通＞
（１）カスタムメニューのファイル名設定にRUIZU X02用(「日付_講座名」)のプリセット追加
　　　RUIZU X02用プリセットを選択することにより、以下のようになります。
　　　- ファイル名を「YYMMDD_講座名」の形式にします。
　　　- 「レベル１」「レベル２」「入門編」「初級編」「中級編」「応用編」の講座のアルバムを以下の形式にします。
　　　　　「レベル１」：L1_中学生の基礎英語
　　　　　「レベル２」：L2_中学生の基礎英語
　　　　　「入門編」「初級編」「中級編」「応用編」
　　　　　（例）入門_ドイツ語（「まいにち」は削除）
 ※「RUIZU X02」はポータブルMP3プレーヤーです。使用していない方にはこの機能は影響ありません。
    
（２）同一放送日に異なる配信がある番組への対応
　　　- ファイル名の日付の後に「-1」「-2」「-3」を付けるようにしました。
　　　- ファイル名の最後など「日付の後」以外に「-1」「-2」「-3」を付毛たい場合は「%i」で設定できます。
　　　- 「_1」「_2」「_3」（「-」に代えて「_」(アンダーバー)）にしたいときは「%_%i」で設定できます。

※ゴガク講座には対象となる番組はありません。

（３）【任意らじる聴き逃し番組設定】の「その他１」内の「アラビア語」を「Asian View」に置き換えました。
※2024/03現在「Asian View」の土曜日放送が上記（２）の対象番組です。

（４）その他bug-fix

＜macoS共通＞
（１）stylesheet（qssファイル）の格納場所の仕様が反映されていなかったので、反映しました。
　　　※　qssファイルを書き換えたい方以外、通常の使い方では関係ありません。

※MacOS版は動作確認していません。
※今回の更新は必須ではありませんが、2023/4以前の版を使用している方、2024/04以降も使用する方は更新を検討ください。

<hr /><em>This discussion was created from the release <a href='https://github.com/CSReviser/CaptureStream2/releases/tag/20240306'>RUIZU用(「日付_講座名」)のプリセット追加 、他</a>.</em>

**Comment by acchi3** [comment]
いつもありがとうございます。
3/21版の[macOS-14版】でテストしました。
Xの方にもコメントしましたが、ファイル名にしていている%Yが2023に変換されました。
動作はしていますが、可能でしたら対応していただけるとうれしいです。

**Reply by CSReviser** [reply]
動作確認下さりありがとうございます。
不具合現象が確認出来ておりません。
確認された番組名や日付をお知らせ下さい。

**Reply by CSReviser** [reply]
不具合現象を確認出来ました。[前週]にチェックがある場合の問題でした。次回改版時に修正します。

**Reply by acchi3** [reply]
対応ありがとうございました。4/6版で確認できました。
